### DISCUSSION TITLE
[Feature request]: new tempate: maturin project for pyo3 python extension module

### DISCUSSION THREAD

**Discussion by evan0greenup** [intellij-rust/intellij-rust#10038] [IntelliJ Rust Plugin Development]
It would be nice to add support for PyO3 maturin project support. Which automatically create a new virtualenv and install maturin in it and create files which is same as `maturin init`.

This can make Rust Python extension module development much easier.

**Comment by dpgraham4401** [comment]
Anyone working on this?
